README.md

```diff
@@ -39,12 +39,15 @@ Then go into Plugins > napari-cellseg3d, and choose which tool to use.
 
 ## News
 
-**New version : v0.1.2**
+**New version : v0.1.3**
 
-- Fixed manifest issue for PyPi
+- Small API changes for training/inference from a script
+- Changed project name to "napari_cellseg3d" to avoid setuptools deprecation
```

Comment:
a name change should not be a patch, this should be at min, a minor change, so bump to 0.2

Reply:
Okay, perhaps we can discuss it in #71 ?